Applications/Games/Heroes of Might & Magic IV/Local (Gold Edition)/script.js

```diff
@@ -0,0 +1,48 @@
+const LocalInstallerScript = include("engines.wine.quick_script.local_installer_script");
+const Resource = include("utils.functions.net.resource");
+const {LATEST_DEVELOPMENT_VERSION} = include("engines.wine.engine.versions");
+const Downloader = include("utils.functions.net.download");
+
+new LocalInstallerScript()
+    .name("Heroes of Might & Magic IV")
+    .editor("New World Computing, Inc.")
+    .applicationHomepage("")
+    .author("Zemogiter")
+    .category("Games")
+    .wineDistribution("upstream")
+    .wineVersion(LATEST_DEVELOPMENT_VERSION)
+    .executable("H4mod.exe")
+    .postInstall(function (wine) {
+        const dataDirectory = wine.prefixDirectory() + "drive_c/" + wine.programFiles() + "Ubisoft/Heroes of Might & Magic IV - Złota Edycja/Data";
+
+        const patchForGameExecutable = new Resource()
+            .wizard(this._wizard)
```

Comment:
This should not work.

The wizard can be accessed using `wine.wizard()`.

Reply:
You are missing the `()`

Reply:
where?

Reply:
You wrote `wine.wizard` instead of `wine.wizard()`